flink-end-to-end-tests/test-scripts/test_glue_schema_registry_json.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
```

Comment:
I think this script can be deleted too now we have migrated to the Java e2e tests?

Reply:
Deleted in the latest commit.